addons/account_peppol/models/account_move.py

```diff
@@ -63,6 +64,22 @@ def _compute_peppol_move_state(self):
             else:
                 move.peppol_move_state = move.peppol_move_state
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        moves = super().create(vals_list)
+        auto_invoice = self.env['ir.config_parameter'].sudo().get_param('sale.automatic_invoice')
+        if not str2bool(auto_invoice) or 'website_id' not in self._fields:
+            return moves
+        for move in moves:
+            if (
+                move.is_sale_document(include_receipts=True)
+                and move.website_id
+            ):
+                move.commercial_partner_id.button_account_peppol_check_partner_endpoint(
+                    company=move.company_id,
+                )
```

Comment:
You're going to return the `moves` anyway, the early return just made this a bit confusing. Let's rewrite it like this:
```suggestion

        if auto_invoice and 'website_id' in self._fields:
            for move in moves:
                if move.is_sale_document(include_receipts=True) and move.website_id:
                    move.commercial_partner_id.button_account_peppol_check_partner_endpoint(company=move.company_id)

        return moves